### PR TITLE
Use `on conflict do nothing` query from postgres 9.5

### PR DESCRIPTION
**Pull request by om-sharma**
It enables us to remove the in query on item table which was getting slow on big table.

### CODE REVIEW

File: src/test/java/uk/gov/functional/PGFunctionalTest.java [hunk 1]
```diff
@@ -107,6 +107,36 @@ public void loadTwoSameItems_addsTwoRowsInEntryAndOnlyOneRowInItemTable() {
...
+        assertThat(r.getStatus(), equalTo(204));
+
+        String canonicalItem1 = new String(canonicalJsonMapper.writeToBytes(canonicalJsonMapper.readFromBytes(item1.getBytes())));
+        String canonicalItem2 = new String(canonicalJsonMapper.writeToBytes(canonicalJsonMapper.readFromBytes(item2.getBytes())));
+
+        List<String> entries = testEntryDAO.getAllHex();
+        assertThat(entries.size(), equalTo(3));
+        assertThat(entries.get(0), equalTo(DigestUtils.sha256Hex(canonicalItem1)));
+        assertThat(entries.get(1), equalTo(DigestUtils.sha256Hex(canonicalItem1)));
+        assertThat(entries.get(2), equalTo(DigestUtils.sha256Hex(canonicalItem2)));
```
Comment (philandstuff) [inline src/test/java/uk/gov/functional/PGFunctionalTest.java]: you can use the (badly-named) `contains()` matcher for this

**Comment by philandstuff** [comment]
needs rebase but otherwise 👍

File: src/test/java/uk/gov/functional/PGFunctionalTest.java (hunk 1 above)
Comment (om-sharma) [inline src/test/java/uk/gov/functional/PGFunctionalTest.java]: updated

**Comment by om-sharma** [comment]
Rebased!!

**Pull request merged**